common/panic-log/src/lib.rs

```diff
@@ -8,7 +8,6 @@ use cortex_m::asm;
 fn panic(info: &PanicInfo) -> ! {
     log::error!("{}", info);
 
-    loop {
-        asm::bkpt()
-    }
+    // abort instruction: triggers a HardFault exception which causes probe-run to exit
```

Comment:
this may be splitting hairs, just asking in case the question comes up in the workshop: according to my understanding of https://docs.rs/cortex-m/0.6.3/cortex_m/asm/fn.udf.html and https://developer.arm.com/documentation/dui0801/g/A32-and-T32-Instructions/UDF `udf()` causes an *Undefined Instruction* Exception which is then escalated to a HardFault– is saying "HardFault exception" the shirthand for this whole chain of events or am I missing something?

Reply:
> is saying "HardFault exception" the shorthand for this whole chain of events 

yes